src/Sylius/Component/Cart/Provider/CartProviderInterface.php

```diff
@@ -14,39 +14,14 @@
 use Sylius\Component\Cart\Model\CartInterface;
 
 /**
- * Interface for object that is accessor for cart.
- * It should retrieve existing cart or create new one based on storage.
+ * Used by context to obtain a cart.
  *
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
 interface CartProviderInterface
```

Comment:
Btw. it looks like it has the same methods as `CartContextInterface`, can we drop the concepts of providers and just use composition in order to create contexts that relies on another contexts? (Just like CompositeChannelContext, CompositeCurrencyContext in #5497, it will make all our contexts using the same approach).